themes/_core/js/product.js

```diff
@@ -324,4 +324,14 @@ $(document).ready(() => {
             args.product_url
         );
     });
+
+    prestashop.on('updateCart', (event) => {
+        if (!event || !event.reason || event.reason.linkAction != 'add-to-cart' ) {
+          return;
+        }
+        const $productActions = $('.product-actions');
+        const $quantityWantedInput = $productActions.find('#quantity_wanted:first');
```

Comment:
`#quantity_wanted:first` it's an idea, you don't need to add `:first` and select it from `.product-actions`, you need to fix the duplicate id :/

Reply:
I actually simply copied the existing code, but indeed the `:first` is useless since there is only one element with this id. I corrected it in other places as well.